### ISSUE TITLE
Integration with Prettier VSC extension

### ISSUE THREAD

**Issue by sammichelson** [ember-template-lint/ember-template-lint-plugin-prettier#216]
Wondering if we could integrate ember-template-lint w/ the VSC Prettier extension.  Thanks!

esbenp.prettier-vscode https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

**Comment by NullVoxPopuli** [comment]
> out of the box

Don't you, at a minimum, still have to have a specific config / .prettierrc.js with something telling it to do stuff?

**Comment by sammichelson** [comment]
> @sammichelson I think the VSC Prettier extension is supposed to format handlebars code "out of the box". Do you have a recent version of Prettier installed in your project? Is there any `prettierignore` in your project?
> 
> Or maybe you want to precise your question?

I actuallly don't think this is exclusive to ember-template-lint. Looks like the Prettier extension will auto-format syntax even if the specific rule tied to that syntax issue is turned off in .eslintrc.js or template-lintrc.js. 

On a side note, I also noticed the Stable Ember Language Server extension works well with showing linting errors toggled on or off in template-lintrc.js but it doesn't work with the todo functionality of ember-template-lint. So for rules made into todos, Stable Ember Lanuage will still show those rules as errors when they should show as warnings etc. I was told that this is an issue with ember-template-lint not giving Stable Ember Language the right information.

**Comment by dcyriller** [comment]
> Looks like the Prettier extension will auto-format syntax even if the specific rule tied to that syntax issue is turned off in .eslintrc.js or template-lintrc.js.

eslint and ember-template-lint on one side and prettier on the other side are different tools. So if you wish to configure how Prettier behaves, you should do so with the .prettierrc ([see this link for details](https://prettier.io/docs/en/configuration.html)) and the .prettierignore.

For instance if you wish to disable Prettier for hbs files, you can write something like `**/*.hbs` in .prettierignore. Then I would expect the prettier extension for VSCode to read these config files and don’t format your hbs files.

Btw, nowadays I don’t really use eslint-plugin-prettier and this plugin anymore. I have the following script `yarn lint` script in CI (I find this config to be simple):
```
  "scripts": {
    "lint": "npm-run-all --parallel lint:*",
    "lint:eslint": "eslint . --cache",
    "lint:lockfile": "yarn install && git diff --exit-code",
    "lint:prettier": "prettier . --check",
    ....
  },
```
